.github/workflows/azure-app-service-api.yml

```diff
@@ -27,6 +27,10 @@ jobs:
       - name: Build with dotnet
         working-directory: ./Wordle.Api
         run: dotnet build --configuration Release
```

Comment:
Features

Create an editor for the available words ✅
Create a page named "WordEditor" that allows words to be edited ✅
Support deleting a word ✅
Support adding a new word ✅
Editing words is not supported ✅ 
Duplicate words should not be allowed ✅
Support changing the common word boolean flag ✅
Words should be sorted alphabetically ✅
Provide a text box for searching words AS YOU TYPE which supports searching for words starting with letters specified ✅
Provide a menu item for navigating to the WordEditor page ✅
Anyone can look at the word list ✅
Word list is paginated (10-100 per page) ✅
Any logged in user can change the common word flag ❌✅ (I think you guys have this, but I am not sure; I wasn't able to log in so I couldn't check this one)
There is a login page/component/popup/etc for users to sign in ✅
Only users over 21 years of age (based on birthday) with a claim of MasterOfTheUniverse can add and remove words ❌✅ (I tried making an account but it didn't let me sign up; I think maybe adding requirements for accounts)
Users can be set up at start up in the API project as a seed (apriori), there does not need to be a sign up mechanism ✅




Extra Credit (I don't think the extra credit was attempted? If I am wrong please let me know, so I can check it out!)

Find a bug in the application, create an issue, submit a pull request against the issue
Add the ability to sign up for an account by inputting an email and password
Allow the user to elevate their level of privilege by inputting a secret phrase.
Select pagination record count

Good job, I liked navigating through your site and the background looks cool!

Reply:
I was able to create a new "user" and log in.
You will need to have @ somewhere in your username
your password needs a capital letter, a lower case letter, a symbol, and a number.

I don't think the policy stuff is working atm tho..

Reply:
We got the policy stuff working! It should be up now.